README.md

```diff
@@ -391,6 +391,10 @@ entire form. With a prefix argument do not thread the last form.
 `clojure-unwind-all`: Fully unwind a threaded expression removing the threading
 macro.
 
+`clojure-cycle-privacy`: Cycle privacy of `def`s or `defn`s. Use metadata
+explicitly with setting `clojure-use-metadata-for-privacy` to `t` for `defn`s
```

Comment:
clojure-ts-...

Reply:
Oh, I need to fix it for all refactoring commands in README :) Copy-paste :)

Reply:
I figured as much - btw, you forgot to update `clojure-use-metadata-for-privacy`.

Reply:
Fixed.